Source/core/Rectangle.h

```diff
@@ -21,7 +21,9 @@
 #define __RECTANGLE_H
 
 #include <algorithm>
+#ifdef WIN32
```

Comment:
Didn't try to build it but had a quick check at the code, is the xutility needed anyway for the windows build?

Reply:
Also since the windows build is capable of running 32 bits and 64 bits, make sure to use the proper pre-processor statement (__WINDOWS__) unless this header file is only required on 32 Bits builds and not on 64 buts, which I doubt :-).